src/inverseRobot.js

```diff
@@ -25,6 +25,20 @@
  */
 function inverseRobot(robot) {
   // write code here
+  let inverseNewRobot = {};
+
+  for (const key in robot) {
+    const val = robot[key];
+    const newKey = val;
+
+    if ((newKey in inverseNewRobot) === true) {
+      inverseNewRobot = null;
+      break;
```

Comment:
```suggestion
      return null;
```
you can just return from function here